app/components/ExpoAssets/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react"
+import { FlatList, Image, View, ImageStyle, TextStyle, ViewStyle } from "react-native"
+import { assets, ImageAsset } from "./assets"
+import { ExpoDemoCard, Text } from ".."
+
+const renderImage = ({ item }: { item: ImageAsset }) => {
+  return (
+    <View style={$imageContainer}>
+      <Text style={$text}>{item.directory}</Text>
+      <Text style={$text}>{item.name}</Text>
+      <Image source={item.image} height={$image.height} width={$image.width} style={$image} />
+    </View>
+  )
+}
+
+const imageKey = (item: ImageAsset) => `${item.directory}/${item.name}`
+
+export const ExpoAssets: () => JSX.Element = () => {
+  return (
+    <ExpoDemoCard
+      title="Image Assets"
+      ContentComponent={
+        <FlatList data={assets} keyExtractor={imageKey} renderItem={renderImage} numColumns={3} />
+      }
+    />
+  )
+}
+
+const $image: ImageStyle & { width: number; height: number } = {
```

Comment:
I've never seen this style of variable naming

Reply:
This demo was created using Ignite boilerplate -- this is the way they name style objects.